websockets/params.go

```diff
@@ -51,8 +53,24 @@ func buildParams(state *lib.State, rt *goja.Runtime, raw goja.Value) (*wsParams,
 			if v, ok := jarV.Export().(*httpModule.CookieJar); ok {
 				parsed.cookieJar = v.Jar
 			}
+		case "compression":
+			// deflate compression algorithm is supported - as defined in RFC7692
+			// compression here relies on the implementation in gorilla/websocket package, usage is
+			// experimental and may result in decreased performance. package supports
+			// only "no context takeover" scenario
+
+			algoString := strings.TrimSpace(params.Get(k).ToString().String())
+			if algoString == "" {
```

Comment:
Is there a particular reason we want to support empty algorithim :thinking:

Reply:
Not at all. Good catch :+1: Let me adjust this

Reply:
Resolved in f4697c4